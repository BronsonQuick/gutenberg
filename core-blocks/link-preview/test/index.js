/**
 * Internal dependencies
 */
import { name, settings } from '../';
import { blockEditRender } from '../../test/helpers';

describe( 'core-embed/link-preview', () => {
	test( 'block edit matches snapshot', () => {
		const wrapper = blockEditRender( name, settings );

		expect( wrapper ).toMatchSnapshot();
	} );
} );
