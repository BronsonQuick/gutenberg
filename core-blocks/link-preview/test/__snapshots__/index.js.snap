// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core-embed/link-preview block edit matches snapshot 1`] = `
<div
  class="components-placeholder wp-block-embed"
>
  <div
    class="components-placeholder__label"
  >
    Link preview
  </div>
  <div
    class="components-placeholder__fieldset"
  >
    <form>
      <input
        aria-label="Link preview"
        class="components-placeholder__input"
        placeholder="Enter URL hereâ€¦"
        type="url"
        value=""
      />
      <button
        class="components-button is-button is-default is-large"
        type="submit"
      >
        Preview
      </button>
    </form>
  </div>
</div>
`;
